src/test/java/org/jabref/logic/importer/fetcher/CiteSeerTest.java

```diff
@@ -0,0 +1,36 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibtexEntryTypes;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@FetcherTest
+class CiteSeerTest {
+
+    CiteSeer fetcher;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        fetcher = new CiteSeer();
+    }
+
+    @Test
+    void searchByQueryFindsEntry() throws Exception {
+        BibEntry expected = new BibEntry();
+        expected.setType(BibtexEntryTypes.MISC);
+        expected.setField("author", "Wang Wei and Zhang Pingwen and Zhang Zhifei");
+        expected.setField("title", "Rigorous Derivation from Landau-de Gennes Theory to Eericksen-leslie Theory");
+        expected.setField("doi", "10.1.1.744.5780");
```

Comment:
We need test for all other parsed fields as well!
- JOURNAL
- YEAR
- VOLUME
- PAGES
- ISSUE

Reply:
I was not able to find an entry that had pages/volume or issue information. However, I added a test covering journal and year.